source/builders.txt

```diff
@@ -16,7 +17,7 @@ the CRUD API:
 .. TODO replace with links
 
 - Projections: Support for building projections
-- Sorts: Support for building sort criteria
+- :ref:`scala-builders-sorts`: Support for building sort specifications
```

Comment:
Suggestion:

In the first section of the content, it looks like the term "sort criteria operators" is used and no references are made to "specifications". I think it would be more clear to use the term introduced in the content.